app/assets/stylesheets/user_analytics.scss

```diff
@@ -746,4 +746,61 @@ select.form-control, input.form-control {
     box-shadow: 0 2px 1px rgba(0,0,0,.16);
     box-sizing: border-box;
   }
-}
\ No newline at end of file
+}
+
+.switch {
```

Comment:
This is the "did you redistribute?" switch? Could we add a screenshot to PR desc?